public/app/plugins/datasource/graphite/configuration/ConfigEditor.tsx

```diff
@@ -83,7 +83,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
           dataSourceConfig={options}
           onChange={onOptionsChange}
         />
-        {config.featureToggles.secureSocksDatasourceProxy && (
+        {config.secureSocksDSProxyEnabled && (
```

Comment:
Nit. Shouldn't this set the secureSocksDSProxyEnabled prop on DataSourceHttpSettings instead?

Reply:
Or do we always support this with Graphite?

Reply:
Good catch! It should use that. Updated :)